lib/airbrake-ruby.rb

```diff
@@ -84,6 +84,14 @@ module Airbrake
   #  special cases where we need to work around older implementations
   JRUBY = (RUBY_ENGINE == 'jruby')
 
+  # Defined here so they can be used multiple times.
+  ruby_version_27 = Gem::Version.new('2.7')
+  ruby_version = Gem::Version.new(RUBY_VERSION)
+
+  # @return [Boolean] true if this Ruby supports safe levels and tainting,
+  #  to guard against using deprecated or unsupported features.
+  HAS_SAFE_LEVEL = (RUBY_ENGINE == 'ruby' && ruby_version < ruby_version_27)
```

Comment:
I would just inline the variables since they don't make the code any clearer (and the comment on line 87 is redundant).

```suggestion
  HAS_SAFE_LEVEL = (
    RUBY_ENGINE == 'ruby' && 
    Gem::Version.new(RUBY_VERSION) < Gem::Version.new('2.7')
  )
```

Reply:
OK.